dnf/cli/commands/repoquery.py

```diff
@@ -120,6 +120,8 @@ def set_argparser(parser):
                                       help=_("check non-explicit dependencies (files and Provides); default"))
         whatrequiresform.add_argument("--exactdeps", action="store_true",
                                       help=_('check dependencies exactly as given, opposite of --alldeps'))
+        parser.add_argument('--deplist', action='store_true', help=_(
+            "list of all dependencies and what packages it provides"))
```

Comment:
"It" refers to "list" ("list" is the subject of the sentence). But a "list" does not "provide" packages. I'm not sure what exactly the sentence was supposed to convey, so I don't know how to fix it, but it needs to be changed.